src/main/resources/application.properties

```diff
@@ -0,0 +1,44 @@
+spring.application.name=ozone-demo-data
+
+# ----------------------------------------------------------------------------------------------------------------------
+# OpenMRS configuration
+# ----------------------------------------------------------------------------------------------------------------------
+openmrs.serverUrl=http://localhost:8080/openmrs
```

Comment:
Shouldn't all these configs have placeholders so that the configurations as be set via env variables?

Reply:
This "http://localhost:8080/openmrs" is already a placeholder. As it is Spring Boot allows you to set env var as `OPENMRS_SERVERURL`